src/pixelator/report/models/annotate.py

```diff
@@ -0,0 +1,160 @@
+"""Copyright © 2023 Pixelgen Technologies AB."""
+
+from __future__ import annotations
+
+from typing import Optional
+
+import pydantic
+
+from pixelator.report.models.base import SampleReport
+from pixelator.report.models.summary_statistics import SummaryStatistics
+
+
+class AnnotateSampleReport(SampleReport):
+    """Model for report data returned by the annotate stage."""
+
+    components_modularity: float
+    fraction_molecules_in_largest_component: float
+    fraction_pixels_in_largest_component: float
```

Comment:
They will be a `pydantic.Field` field as well. It is just not needed to add that explicitly because the `description` option is not passed in. I have updated these field to also have a description